Neos.Flow/Classes/Mvc/DispatchComponent.php

```diff
@@ -105,15 +106,16 @@ protected function prepareActionRequest(ComponentContext $componentContext)
             $httpRequest = $httpRequest->withParsedBody($parsedBody);
         }
 
+        $arguments = ArgumentsHelper::buildUnifiedArguments($httpRequest->getQueryParams(), $parsedBody, UploadedFilesHelper::untangleFilesArray($_FILES));
```

Comment:
Not sure it's good to refer to $_FILES here again. Can we extract that info back from PSR7 UploadedFiles? Otherwise maybe store it in an Request attribute?